src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs

```diff
@@ -132,5 +132,11 @@ public string[] GetFiles(string path, string searchPattern, SearchOption searchO
         {
             return Directory.GetFiles(path, searchPattern, searchOption);
         }
+
+        /// <inheritdoc/>
+        public void Delete(string path)
```

Comment:
Can you change this to TryDelete with bool return, In case file does not exists this API will throw.